src/data.js

```diff
@@ -14115,6 +14115,22 @@ module.exports = [
       'Node.js',
     ],
   },
+  {
+    name: 'Hugh He',
+    description: 'FOSS Developer, Student',
+    url: 'https://github.com/plushugh/plushugh/blob/main/USES.md',
```

Comment:
Hey! The 'url' pattern requires 'use', 'uses', 'using', 'setup', or starting with 'https://gist.github.com/'. Consider updating the URL.

Reply:
Thanks, will add a correct link in a moment